atc/exec/var_step.go

```diff
@@ -0,0 +1,137 @@
+package exec
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"code.cloudfoundry.org/lager"
+	"code.cloudfoundry.org/lager/lagerctx"
+	"sigs.k8s.io/yaml"
+
+	"github.com/concourse/baggageclaim"
+	"github.com/concourse/concourse/atc"
+	"github.com/concourse/concourse/atc/exec/artifact"
+	"github.com/concourse/concourse/atc/exec/build"
+	"github.com/concourse/concourse/atc/worker"
+	"github.com/concourse/concourse/vars"
+)
+
+// SetPipelineStep sets a pipeline to current team. This step takes pipeline
```

Comment:
```suggestion
// VarStep loads a value from a file and sets it as a build-local var.
```

(We'll just amend this later when we add more functionality.)